### ISSUE TITLE
Search box

### ISSUE THREAD

**Issue by dexturr** [vegaprotocol/frontend-monorepo#15]
Implement a debounced search function that follows the following algorithm

if number -> redirect to block/:id
if query is not of length 64 or 66 -> show message that this id looks dodgy
strip 0x from front of query if present
run query on data node
  if any entity is found with that id -> redirect to [entity]/:id
else -> redirect to txs/:id

**Comment by dexturr** [comment]
Blocked by https://github.com/vegaprotocol/data-node/issues/347

**Comment by asiaznik** [comment]
I've reopened the underlying issue because: https://github.com/vegaprotocol/vega/issues/5709

**Comment by asiaznik** [comment]
It appears that the querying works fine on `devnet` - I've initially checked it on `testnet` 😕

**Comment by dexturr** [comment]
Superseded by #1411
